ktor-server/jvm/src/io/ktor/server/routing/RoutingApplicationCall.kt

```diff
@@ -17,6 +18,7 @@ import io.ktor.util.*
 public class RoutingApplicationCall(
     private val call: ApplicationCall,
     public val route: Route,
+    private val job: Job,
```

Comment:
Should we add `CoroutineScope` as a parent to `ApplicationCall` instead?

Reply:
I think it will not work in the case of `NettyApplicationCall` that does not have a context for the current call, but instead, it has a `responseWriteJob` that is created and bound to the call.

Please, feel free to correct me if I'm wrong, but if you look at `NettyHttp1ApplicationCall` there is a (`context: ChannelHandlerContext` which does not provide a CoroutineContext), `engineContext` and `userContext` that are not bound to a specific call. So, we can't get a context for it, but we can easily access a `responseWriteJob` directly from `NettyApplicationCall`.